Networking.xcodeproj/xcshareddata/xcschemes/Networking-mac.xcscheme

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Slightly petty but could we use "Networking iOS", "Networking macOS", "Networking watchOS", "Networking tvOS"? So they're all named after the platform OSs consistently.